### ISSUE TITLE
[ENHANCEMENT]Decimal numbers on Axis

### ISSUE THREAD

**Issue by ncolton33** [dm-p/powerbi-visuals-smlc#17]
**Is your feature request related to a problem? Please describe.**
    Yes, currently the chart will not render if using decimal values on the Axis. 

    I'm trying to plot decimal values on the y-axis (distance (feet) ) and decimal values on the x-axis (total time elapsed (days or hours) ) with a text category for Small Multiple. The chart does not render any results. The issue doesn't appear to be the data limit, as I've tried limiting the values to be well within the threshold of and the (i) is not showing up at the top.

**Describe the solution you'd like**
    I'd like the added functionality of using decimal numbers on Axis and Values. This appears to already be supported based on the docs, so maybe this is a bug? Sorry if this is in the wrong place.

A clear and concise description of what you want to happen. If you have mockups or something to attach that will help to communicate your idea then that will be massively helpful :)
    I'll add an screenshot of a similar solution in Spotfire with some dummy data.
![example_line](https://user-images.githubusercontent.com/26281702/79033030-8a981e00-7b70-11ea-95cc-2fe1ad6e425f.PNG)

**Additional context**

**Comment by dm-p** [comment]
Hi @ncolton33, and thanks for your feedback.

I suspect that this is a duplicate of #6 with linear axes starting from 0 (which is a bug). This is admittedly a pretty terrible oversight on my part. It will be fixed in the next version which I'm going to submit to the Marketplace as soon as possible - I'm just waiting on another issue in the Microsoft tooling to be fixed before I submit, and have just chased them today for confirmation on when this will be resolved.

There's a [workaround documented on that issue](https://github.com/dm-p/powerbi-visuals-smlc/issues/6#issuecomment-589833084) which may help in the interim, so if you might be able to confirm if you're able to apply this and it works for you then that would be a great help in making that there's nothing else going on here that we need to look at.

**Comment by ncolton33** [comment]
@dm-p Thank you for such a quick response. Yes, that workaround did solve the problem. I'll run with this in the interim and eagerly await your next release. Thanks again!

**Comment by dm-p** [comment]
Thanks for confirming, @ncolton33! FYI the latest version has been submitted to the marketplace - it's difficult to provide an ETA (it can often take weeks to get through to production as the team have a number of checkpoints), but when you see 2.1.0 or higher in your reports the issue will be resolved.
